nixos/lib/make-options-doc/mergeJSON.py

```diff
@@ -57,19 +57,24 @@ def _get_method(self, name):
             try:
                 return super(Renderer, self)._get_method(name)
             except AttributeError:
-                def not_supported(children, **kwargs):
-                    raise NotImplementedError("md node not supported yet", name, children, **kwargs)
+                def not_supported(*args, **kwargs):
+                    raise NotImplementedError("md node not supported yet", name, args, **kwargs)
                 return not_supported
 
         def text(self, text):
             return escape(text)
         def paragraph(self, text):
             return text + "\n\n"
+        def newline(self):
+            return "<literallayout>\n</literallayout>"
         def codespan(self, text):
-            return f"<literal>{text}</literal>"
+            return f"<literal>{escape(text)}</literal>"
         def block_code(self, text, info=None):
-            info = f" language={quoteattr(info)}" if info is not None else ""
-            return f"<programlisting{info}>\n{text}</programlisting>"
+            if info == "{=docbook}":
```

Comment:
not sure having that hatch is such a great thing. we'll have to make some adjustment to the documentation to get it to work as markdown anyway, and if we have a hatch we have one more place to keep in mind when we finally axe docbook.

do you have a special use in mind where this is necessary? maybe a few more extension to the renderer would be better.

Reply:
I used it while I couldn't get `blockquote` accepted by the docbook tooling that runs after this. I suppose I could have used a plain old docbook description instead. I'll remove it.